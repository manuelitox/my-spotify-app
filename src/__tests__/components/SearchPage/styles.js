import React from 'react'
import { shallow } from 'enzyme'

import StyledSearchPage from 'components/SearchPage/styles'

describe('Component | StyledSearchPage', () => {
  let component 
  beforeEach( () => {
    component = shallow(
      <StyledSearchPage />
    )
  })
  it('should render a styled component', () => {
    expect(component).toMatchSnapshot()
  })
})