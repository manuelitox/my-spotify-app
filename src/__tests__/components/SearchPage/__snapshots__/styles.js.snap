// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component | StyledSearchPage should render a styled component 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": false,
        "rules": Array [
          "
  background-image: linear-gradient(#3DC9C2, #3D4EC9);
  background-size: cover;
  background-position: center;
  color: white;
  height: 70vh;
  text-align: center;
  padding: 180px 0;
  width: 100%;
  @media only screen and (max-width: ",
          [Function],
          ") {
    padding: 80px 0;
  }
",
        ],
      },
      "displayName": "styled.div",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
